hypothesis-python/src/hypothesis/provisional.py

```diff
@@ -29,6 +29,17 @@
 import string
 
 import hypothesis._strategies as st
+from hypothesis.errors import InvalidArgument
+
+
+TOP_LEVEL_DOMAINS = []
+# This file is sourced from http://data.iana.org/TLD/tlds-alpha-by-domain.txt
+# The file contains additional information about the date that it was last updated.
+with open('./vendor/tlds-alpha-by-domain.txt') as tld_file:
```

Comment:
It's possible that this will fail for some reason, so I'd like to have short list of TLDs in the source code that we can use if e.g. a redistributor hasn't included the text file.

The path should also be given relative to `__file__`, not `.`.